rust-runtime/aws-smithy-json/src/serialize.rs

```diff
@@ -309,7 +309,7 @@ mod tests {
         object.finish();
 
         assert_eq!(
-            r#"{"epoch_seconds":5.2,"date_time":"2021-05-24T15:34:50.123Z","http_date":"Wed, 21 Oct 2015 07:28:00 GMT"}"#,
+            r#"{"epoch_seconds":5.2,"date_time":"2021-05-24T15:34:50Z","http_date":"Wed, 21 Oct 2015 07:28:00 GMT"}"#,
```

Comment:
We should change this test to put a fraction in http_date—and I think we need to back out the changes to date_time, only http_date omits the subsecond. My bad on that—good catch from @jdisanti